lib/v2/arcteryx/regear-new-arrivals.js

```diff
@@ -0,0 +1,51 @@
+const got = require('@/utils/got');
+const cheerio = require('cheerio');
+const { art } = require('@/utils/render');
+const path = require('path');
+
+const host = 'https://www.regear.arcteryx.com';
+function getUSDPrice(number) {
+    return (number / 100).toLocaleString('en-US', { style: 'currency', currency: 'USD' });
+}
+module.exports = async (ctx) => {
+    const url = `${host}/shop/new-arrivals`;
+    const response = await got({
+        method: 'get',
+        url,
+    });
+
+    const data = response.data;
+    const $ = cheerio.load(data);
+    const contents = $('script:contains("window.__PRELOADED_STATE__")').text();
+    const regex = /{.*}/;
+    const items = JSON.parse(contents.match(regex)[0]).shop.items;
+    const list = items.map((item) => {
+        const data = {};
+        data.title = item.displayTitle;
+        data.link = item.pdpLink.url;
+        data.imgUrl = JSON.parse(item.imageUrls).front;
+        data.availableSizes = item.availableSizes;
+        if (data.availableSizes.length === 0) {
+            return data;
+        }
```

Comment:
Please use [.filter()](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/filter) as the empty item is still in the feed as you can see from https://github.com/DIYgod/RSSHub/pull/12063#issuecomment-1459685143